src/scripts/main.js

```diff
@@ -1,8 +1,33 @@
 'use strict';
 
 const people = require('./lib/people');
+const table = document.querySelector('.dashboard');
 
-// eslint-disable-next-line no-console
-console.log(people); // you can remove it
+for (const person of people) {
+  const tableRow = document.createElement('tr');
+  const nameCell = document.createElement('td');
+  const genderCell = document.createElement('td');
+  const bornCell = document.createElement('td');
+  const diedCell = document.createElement('td');
+  const ageCell = document.createElement('td');
+  const centuryCell = document.createElement('td');
 
-// write your code here
+  nameCell.textContent = person.name;
```

Comment:
```suggestion
  const { name, born, sex, died, born } = person;
  
  nameCell.textContent = name;
```
small advice - you can destructure `person` object to not repeat `person.` each time